community/sample-apps/firebase-jwt/app/index.html

```diff
@@ -5,7 +5,7 @@
   <body>
     <h1> Firebase Auth + Hasura JWT example </h1>
     <form id="login-form">
-      Email: <input id="email" type="text"/>
+      Email: <input id="email" type="emailaddress"/>
```

Comment:
@Harsh2803 thank you for your contribution 🙂 as far as I know, `emailaddress` doesn't exist as an HTML input type. It should be `email`: https://www.w3schools.com/tags/att_input_type_email.asp. 
Could you make the change?